sktime/util.py

```diff
@@ -266,6 +266,91 @@ def call_member(obj, f, *args, **kwargs):
     return method
 
 
+class QuantityStatistics(object):
```

Comment:
does this need to be unit tested or meh

Reply:
optimally it would be unit tested, right now it just appears some other tests so one could say it is integration tested at least :)

Reply:
diplomatic, i like it

Reply:
unit tested it now anyways

Reply:
woot